pkgs/build-support/fetchzip/tests.nix

```
@@ -12,6 +12,11 @@ in
   postFetch = testers.invalidateFetcherByDrvHash fetchzip {
     inherit url;
     sha256 = "sha256-7sAOzKa+9vYx5XyndHxeY2ffWAjOsgCkXC9anK6cuV0=";
-    postFetch = ''touch $out/filee'';
+    postFetch = "touch $out/filee";
+  };
+
+  hiddenDir = testers.invalidateFetcherByDrvHash fetchzip {
+    url = "file://${./hiddendir.tar}";
```

```suggestion
    url = "file://${runCommand "hiddendir.tar" {} ''
      mkdir .foo
      tar -cf $out .foo
    ''}";
```

Something like this might be easier to read, especially on GitHub, which doesn't render tarballs nicely.